laravel/app/Models/Post.php

```diff
@@ -15,4 +15,9 @@ public function user()
     {
         return $this->belongsTo('App\Models\User');
     }
+
+    public function replies()
+    {
+           return $this->hasMany('App\Models\replies');
```

Comment:
ケースもしっかりと意識しましょう 👍 (大文字か小文字か）
```suggestion
           return $this->hasMany('App\Models\Replies');
```

Reply:
以後きをつけます！